### PR TITLE
Tania Gayosso

### PR DESCRIPTION
**Pull request by taniagd**
Changes in name of the package according to Google Java Style Guide and constant added.

### CODE REVIEW

**Review by luisalanmz — APPROVED** [review]
LGTM

File: TaniaGayosso/HackerRank_August25/IfElse.java [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+import java.util.Scanner;
+
+public class IfElse {
+
+	private static final Scanner scanner = new Scanner(System.in);
+
+	public static void main(String[] args) {
+
+		final int NUMBER = scanner.nextInt();
```
Comment (luisod78) [inline TaniaGayosso/HackerRank_August25/IfElse.java:11]: NUMBER is a variable (read if from console), not a constant, so it doesn't need to be final

File: TaniaGayosso/HackerRank_August25/IfElse.java [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+		final String MESSAGE;
+		scanner.skip("(\r\n|[\n\r\u2028\u2029\u0085])?");
+
+		if (NUMBER % 2 == 1) {
```
Comment (luisod78) [inline TaniaGayosso/HackerRank_August25/IfElse.java:15]: I don't get it....
If NUMBER==3 then it's Weird... 
so why in line 18 you are evaluting: if (NUMBER >= 2 && NUMBER <= 5) ?? 
hmmmmmm.... something's wrong right?

**Review by luisod78 — CHANGES_REQUESTED** [review]

File: TaniaGayosso/HackerRank_August25/IfElse.java (hunk 1 above)
Comment (taniagd) [reply TaniaGayosso/HackerRank_August25/IfElse.java:11]: Ok, thank you.

File: TaniaGayosso/HackerRank_August25/IfElse.java (hunk 2 above)
Comment (taniagd) [reply TaniaGayosso/HackerRank_August25/IfElse.java:15]: Yes, that's right. I will be fixing that.

File: TaniaGayosso/HackerRank_August25/IfElse.java [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+	private static final Scanner scanner = new Scanner(System.in);
+	public static final String WEIRD_NUMBER = "Weird";
+	public static final String NOT_WEIRD_NUMBER = "Not Weird"; 
+	
+	public static void main(String[] args) {
+
+		int number = scanner.nextInt();
+		scanner.skip("(\r\n|[\n\r\u2028\u2029\u0085])?");
+
+		if (number % 2 == 1 || number >= 6 && number <= 20)  {
```
Comment (luisod78) [inline TaniaGayosso/HackerRank_August25/IfElse.java:16]: If  n is even and in the inclusive range of  to , print Weird

What if number == 6

**Review by luisod78 — CHANGES_REQUESTED** [review]

**Review by luisalanmz — APPROVED** [review]
Just attend Luis comments

**Pull request merged**